pypeit/coadd2d.py

```diff
@@ -1023,35 +1086,28 @@ def compute_offsets(self, offsets):
                 It could be a list of offsets, or a string, or None. If equal to 'maskdef_offsets' the
                 offsets computed during the slitmask design matching will be used.
         """
+        super(MultiSlitCoAdd2D, self).compute_offsets(offsets)
```

Comment:
I would just change this to self.compute_offsets_init(offsets) or something like that. To me it is confusing to overload a method with a method of the same name but still call the parent method.